proto/buf/alpha/registry/v1alpha1/webhook_payload.proto

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/webhook.proto";
+import "google/protobuf/timestamp.proto";
+
+// WebhookListenerService represents a service implemented by a customer, to
+// receive webhook events invocations from the BSR.
+service WebhookListenerService {
```

Comment:
WebhookReceiverService?

Reply:
Sure why not, although I don't see a big difference vs "Listen". What's the upside?

Reply:
What is the expected use of the service definition? Is it so that users can codegen a handler that they give to connect?

In that case, am I expected to provide a `callback_url` that is `https://acme.corp.com/webhooks` and we'll append the `buf.alpha.registry.v1alpha1.WebhookListenerService/ListenWebhook` when making the outgoing request, or should the `callback_url` be `https://acme.corp.com/webhooks/buf.alpha.registry.v1alpha1.WebhookListenerService/ListenWebhook`?

Reply:
The current intention is for it to be `https://acme.corp.com/webhooks`

Reply:
I think that webhook receiving should be possible in languages other than go, given that we have no connect in those other languages it does feel weird that we would enforce `pkg.Service/Method` be part of the url if I'm just writing some python/java/js handler that does the equivalent of `proto.Unmarshal(io.Readall(request.Body()), m)`.

Reply:
They can still implement the gRPC service handler if they're not using connect!